multimodal/src/autogluon/multimodal/utils/onnx.py

```diff
@@ -0,0 +1,98 @@
+import logging
+from typing import Dict, List, Optional, Tuple, Union
+
+from torch import tensor
+
+from ..constants import AUTOMM, FEATURE_EXTRACTION
+
+logger = logging.getLogger(AUTOMM)
+
+
+def get_onnx_input(pipeline: str, config: Optional[Dict] = None):
+    """
+    Get information for a predictor to export its model in onnx format.
+
+    Parameters
+    ----------
+    pipeline
+        The predictor's pipeline.
+    config
+        The predictor's config.
+
+    Returns
+    -------
+    valid_input
+        The valid keys for the input batch.
+    dynamic_axes
+        By default the exported model will have the shapes of all input and output tensors
+        set to exactly match those given in ``args``. To specify axes of tensors as
+        dynamic (i.e. known only at run-time), set ``dynamic_axes`` to a dict with schema:
+            * KEY (str): an input or output name. Each name must also be provided in ``input_names`` or
+              ``output_names``.
+            * VALUE (dict or list): If a dict, keys are axis indices and values are axis names. If a
+              list, each element is an axis index.
+        See torch.onnx.export for more explanations.
+    default_onnx_path
+        The default path of the export onnx model.
+    default_batch
+        The default batch to help trace and export the model.
+
+    """
+    default_onnx_path = None
+    if pipeline == FEATURE_EXTRACTION:
+        valid_input = [
+            "hf_text_text_token_ids",
+            "hf_text_text_valid_length",
+            "hf_text_text_segment_ids",
```

Comment:
Why do we need to add the `hf_text` as the prefix in each key? It looks kind of redundant.

Reply:
It's the default key names in hf input batch. I didn't change it while export onnx.

Reply:
Got it!